src/main/java/teammates/ui/webapi/action/DeleteStudentProfilePictureAction.java

```diff
@@ -0,0 +1,38 @@
+package teammates.ui.webapi.action;
+
+import org.apache.http.HttpStatus;
+
+import com.google.appengine.api.blobstore.BlobKey;
+
+import teammates.common.exception.UnauthorizedAccessException;
+import teammates.common.util.Const;
+
+/**
+ * Deletes a student's profile picture and its picture key.
+ */
+public class DeleteStudentProfilePictureAction extends Action {
+    @Override
+    protected AuthType getMinAuthLevel() {
+        return AuthType.LOGGED_IN;
+    }
+
+    @Override
+    public void checkSpecificAccessControl() {
+        if (!userInfo.isStudent) {
+            throw new UnauthorizedAccessException("Student privilege is required to update this resource.");
+        }
+    }
+
+    @Override
+    public ActionResult execute() {
+        String googleId = getNonNullRequestParamValue(Const.ParamsNames.STUDENT_ID);
+        if (!userInfo.id.equals(googleId) && !isMasqueradeMode()) {
+            return new JsonResult("You are not authorized to delete this student's profile.",
+                    HttpStatus.SC_FORBIDDEN);
+        }
+        String blobKey = getNonNullRequestParamValue(Const.ParamsNames.BLOB_KEY);
```

Comment:
Do you still need this? If you pass in the user ID, you can get the corresponding blob key from the DB.